File: api/src/main/java/net/blueshell/api/dto/EventFeedbackDTO.java
package net.blueshell.api.dto;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = false)
@Schema(name = "EventFeedback")
public class EventFeedbackDTO extends BaseDTO {
    private Long id;
    private String feedback;
    private long eventId;
}

File: api/src/main/kotlin/net/blueshell/api/event/dto/EventFeedbackDTO.kt
package net.blueshell.api.event.dto

import io.swagger.v3.oas.annotations.media.Schema
import net.blueshell.api.shared.dto.AuditedAutoIdDTO

@Schema(name = "EventFeedback")
data class EventFeedbackDTO(
    var feedback: String? = null,
    var eventId: Long = 0
) : AuditedAutoIdDTO()
